CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### `jsonschema-generator`
+#### Added
+- New `Option.FLATTENED_SUPPLIERS` to unwrap the supplied type. `Supplier<T>` would thus be a `type: "T"`.
```

Comment:
Should also add as "Changed":
- that the new `Option` is being included in the `OptionPreset.PLAIN_JSON` by default
- that the `Optional.FLATTENED_OPTIONALS` also considers subtypes of `Optional` (after refactoring it accordingly)